portfolio/src/main/webapp/message-fetch.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * @param {function} fetchObj a mock fetch object for testing purposes
+ */
+const fetchMessage = (fetchObj = null) => {
+    const url = "/data";
```

Comment:
This is where https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes applies